File: Java/LinguagemC/1165 - Numero Primo.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int n, x;
    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> x;
        int c = 0;
        for(int j = 1; j <= x; j++){
            if(x%j==0){
                c++;
            }
        }
        if(c>2){
            cout << x <<" nao eh primo" << endl;
        } else {
            cout << x <<" eh primo" << endl;
        }
    }
 
    return 0;
}

File: Java/JavaScript/1165-NumeroPrimo.js
var input = require("fs").readFileSync('stdin', 'utf8');
var lines = input.split('\n');
var n = parseInt(lines.shift());
for(var i = 0; i < n; i++){
    var x = parseInt(lines.shift());
    var c = 0;
    for(var j = 1; j <= x; j++){
        if(x%j===0){
            c++;
        }
    }
    if(c > 2){
        console.log(x+" nao eh primo");
    } else {
        console.log(x+" eh primo");
    }
}